jar.mn

```diff
@@ -2,6 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+# Just for local testing of in-progress fluent migration
+[localization] en-US.jar:
+  newtab.ftl (./locales-src/newtab.wip)
```

Comment:
Why we are adding newtab.ftl here in jar.mn? won't <link rel="localization" takes care of that if we rename newtab.wip to newtab.ftl

Reply:
I suppose if we aren't actually exporting this branch to m-c until we're done with `newtab.ftl` then we could maybe just directly use `.ftl`. It's just that commits to m-c that modify `.ftl` need extra review but this file is in development and not actually exposed to localizers yet

Reply:
Switched to plain `locales-src/newtab.ftl` and this made me realize if we aren't exporting this branch until we're done, then I'll just copy over locales-src/newtab.ftl into the appropriate m-c/browser/locales/en-US/browser/newtab/newtab.ftl location